custom/data.ts

```diff
@@ -1,73 +1,72 @@
-import { faker } from "@faker-js/faker";
-
 const introduction = {
-  title: "YOUR NAME",
-  text: "And a brief description of you",
+  title: "RAIA",
+  text: "Software Developer • Not a Robot",
 };
 
 const description = {
-  text: faker.lorem.paragraph(),
+  text: `Bringing exploratory mindset to technical leadership,
+        with six years of experience in software development and computational research`,
 };
 
 const projects = [
   {
-    title: faker.lorem.words(2),
-    text: faker.lorem.words(5),
+    title: "Software development",
+    text: "Full stack development description",
     works: [
       {
-        title: faker.word.noun(),
-        text: faker.lorem.sentence(),
-        link: "/",
+        title: "Portfolio",
+        text: "A portfolio project to show my skills and existing projects",
+        link: "https://github.com/raiaqt/betterai-portfolio",
         year: "2024",
         github: true,
```

Comment:
Consider using descriptive keys instead of random generated values for better clarity and maintainability.

Suggested change:
```javascript
 title: "Portfolio",
 text: "A portfolio project to show my skills and existing projects",
 link: "https://github.com/raiaqt/betterai-portfolio",
 year: "2024",
 github: true,
 image: "placeholder.png",
``` 

For reference, see [JavaScript Naming Conventions](https://www.w3schools.com/js/js_conventions.asp).